src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -111,6 +112,12 @@ void GcsResourceManager::HandleGetAllAvailableResources(
 
 void GcsResourceManager::UpdateFromResourceReport(const rpc::ResourcesData &data) {
   NodeID node_id = NodeID::FromBinary(data.node_id());
+  // We only need to update worker nodes' resource usage. Gcs node ifself does not
```

Comment:
You should be able to assert this right? GCS shouldn't ever send out a resource report right?

Reply:
This part is from another feature, which I'll do in the next split PR. So I'll just revert this here.